ansible/roles/networking/tasks/main.yaml

```
@@ -36,6 +36,12 @@
       state: present
       reload: yes
 
+  - name: Flush Iptable rules
+    iptables:
+      flush: true
```

From https://github.com/mesosphere/konvoy/blob/8d174ec0634d2d21728ea457e511de2adc57fa6c/ansible/_flush-iptables.yaml#L9-L16

And CAPPP https://github.com/mesosphere/cluster-api-provider-preprovisioned/blob/7e4c707050a90c33b10471c954ccc211a010b237/preprovisioned/preprovisioned.go#L280-L282
```suggestion
  - name: Flush Iptable rules
    iptables:
      table: "{{ item }}"
      flush: true
    with_items:
      - nat
      - raw
```